uni-resolver-web/src/main/resources/application.yml

```diff
@@ -354,6 +354,7 @@ uniresolver:
       url: ${uniresolver_web_driver_url_did_webvh:http://driver-did-webvh:8080/}
       testIdentifiers:
         - did:webvh:QmPEQVM1JPTyrvEgBcDXwjK4TeyLGSX1PxjgyeAisdWM1p:gist.githubusercontent.com:brianorwhatever:9c4633d18eb644f7a47f93a802691626:raw
+        - did:webvh:QmVJ5nUYb9iugnUz4yDfbe8UFbhmnsvS2EAzSpSfPScRAn:opsecid.github.io
```

Comment:
Are you sure this is a valid DID?  I'm not finding https://opsecid.github.io/ is a GH-Pages site (404) and I'm getting a 404 for https://opsecid.github.io/did.jsonl. @PatStLouis Perhaps it is down until this is published?

I am able to resolve the gist one.

Reply:
@swcurran when the did is published at a bare domain you need to add the .well-known directory:
https://opsecid.github.io/.well-known/did.jsonl